lib/rototiller/utilities/env_var.rb

```diff
@@ -1,10 +1,10 @@
 require 'rototiller/utilities/color_text'
 
 class EnvVar
-  MESSAGE_TYPES = {:nodefault_noexist=>0, :exist=>1, :default_noexist=>2}
+  MESSAGE_TYPES = {:nodefault_noexist=>0, :exist=>1, :default_noexist=>2, :not_required=>3}
   include ColorText
 
-  attr_accessor :var, :message, :default
+  attr_accessor :var, :message, :default, :required
```

Comment:
:nit these need yardoc comments, but they weren't there before, so i'm willing to kick it on down 🎢